osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -501,8 +503,12 @@ private void load(OsuColour colours, AudioManager audioManager, NotificationOver
                     notificationOverlay.Hide();
 
                     volumeOverlay.IsMuted.Value = false;
-                    audioManager.Volume.SetDefault();
-                    audioManager.VolumeTrack.SetDefault();
+
+                    // Check values before resetting, as the user may have only had mute enabled, in which case we might not need to adjust volumes.
+                    if (audioManager.Volume.Value < volume_requirement)
+                        audioManager.Volume.SetDefault();
```

Comment:
Personally, I've never used this notification because it blasts my ears out of my head so I'd prefer if this were setting it to the `volume_requirement` instead or something like that. But alas, I guess it's fine as it is.

Reply:
If we're changing what it sets things to, I'd also want to be updating the default volumes game-wide.